packages/fether-react/src/stores/createAccountStore.js

```diff
@@ -55,11 +56,13 @@ export class CreateAccountStore {
    */
   @action
   clear = async () => {
-    this.jsonString = null;
-    this.parityPhrase = null;
-    this.bip39Phrase = null;
     this.address = null;
+    this.bip39Phrase = null;
+    this.isImport = false;
```

Comment:
I don't think we should reset isImport here, afaik clear() only resets the account info

Reply:
i've pushed commit 1c8f7f2d67d6474a8ce1124f2287869432bf5618